Homebrew/Resolute Spellbook/firstdaybreak-additions.json

```diff
@@ -0,0 +1,370 @@
+[
+    {
+        "name": "Lightning Surge",
+        "level": 2,
+        "school": "Force",
+        "automation": [
+            {
+                "type": "roll",
+                "roll": "2d6",
+                "name": "damage",
+                "higher": {
+                    "3": "1d6",
+                    "4": "1d6",
+                    "5": "1d6",
+                    "6": "1d6",
+                    "7": "1d6",
+                    "8": "1d6",
+                    "9": "1d6"
+                }
+            },
+            {
+                "type": "target",
+                "target": "self",
+                "effects": [
+                    {
+                        "type": "ieffect2",
+                        "name": "Residual Electricity",
+                        "duration": 1,
+                        "attacks": [
+                            {
+                                "attack": {
+                                    "_v": 2,
+                                    "name": "Residual Electricity - Attack",
+                                    "automation": [
+                                        {
+                                            "type": "target",
+                                            "target": "all",
+                                            "effects": [
+                                                {
+                                                    "type": "ieffect2",
+                                                    "name": "Residual Electricity - Effect",
+                                                    "duration": 1,
+                                                    "desc": "Can't make attacks, no reactions"
+                                                },
+                                                {
+                                                    "type": "damage",
+                                                    "damage": "1d6 [lightning]",
+                                                    "higher": {
+                                                        "3": "1d6",
+                                                        "4": "2d6",
+                                                        "5": "3d6",
+                                                        "6": "4d6",
+                                                        "7": "5d6",
+                                                        "8": "6d6",
+                                                        "9": "7d6"
+                                                    }
+                                                }

```

Comment:
this won't work unless they !a residual -l 3 for higher levels. The best way to handle this is to make a variable for the number of die, add that to the effect description on the last line then for the damage node in the attack have ieffect.desc.split()[-1]. See Acidic Strike for an example